hr_expense_analytic_tag/README.rst

```diff
@@ -0,0 +1,90 @@
+=======================================
+Link Analytic Tags with Expense Tracker
+=======================================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fhr-lightgray.png?logo=github
+    :target: https://github.com/OCA/hr/tree/11.0/hr_expense_analytic_tag
+    :alt: OCA/hr
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/hr-11-0/hr-11-0-hr_expense_analytic_tag
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/116/11.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module provides 'Analytic Tags' on Expense. These tags are propagated
+to Analytic Entries
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. Go to expenses form in *Expenses -> My Expenses -> Expenses to Submit* and you will find the new *Analytic Tags* field below *Analytic Account* field
+#. Create a new Expense record with an *Analytic Account* and one or more *Analytic Tags*
+#. Click on *Submit to Manager* button
+#. Click on *Approve* button
```

Comment:
Please @ernestotejeda add field analytic tag in the expense lines of the expense sheet. Thanks

![2018-12-05_22-39-43](https://user-images.githubusercontent.com/8736623/49545874-2f507e80-f8df-11e8-9231-d5bc20746bb5.png)